utils/build-script-impl

```diff
@@ -294,6 +240,17 @@ KNOWN_SETTINGS=(
     skip-local-host-install                       ""                "If we are cross-compiling multiple targets, skip an install pass locally if the hosts match"
 )
 
+components=( cmark foundation libcxx libdispatch libicu llbuild lldb llvm playgroundsupport static-foundation static-libdispatch swift xctest )
```

Comment:
Can we drop the extra spaces around the parens?

Reply:
Sure.